### DISCUSSION TITLE
What is the qxl-like device in cloud-hypervisor?

### DISCUSSION THREAD

**Discussion by qishiyexu** [cloud-hypervisor/cloud-hypervisor#3830] [Q&A] [answered]
Hi, 

Qemu has a qxl device which can be used as a virtual display card, does cloud-hypervisor have similar device? Is this kind of device necessary?

**Comment by sboeuf** [comment, marked as answer]
> Qemu has a qxl device which can be used as a virtual display card, does cloud-hypervisor have similar device?

Cloud Hypervisor doesn't emulate such device. 

> Is this kind of device necessary?

Well it depends the use case you have in mind.

**Reply by qishiyexu** [reply]
Is that mean qemu has a better performance in display?

**Reply by sboeuf** [reply]
It has nothing to do with performance, it's about feature. This is a feature simply not implemented in Cloud Hypervisor.

**Reply by qishiyexu** [reply]
But how does vm created by cloud-hypervisor display output without a graphic card?

**Reply by sboeuf** [reply]
You can use SSH or RDP if you want to connect "graphically" to the guest.

**Reply by qishiyexu** [reply]
Got it. Asked a stupid question.
